sql/_13_issues/_20_2h/answers/cbrd_23665.answer_cci

```diff
@@ -913,7 +913,7 @@ trace
 Trace Statistics:
   SELECT (time: ?, fetch: ?, ioread: ?)
     SCAN (temp time: ?, fetch: ?, ioread: ?, readrows: ?, rows: ?)
-      SCAN (hash temp(h), build time: ?, time: ?, fetch: ?, ioread: ?, readrows: ?, rows: ?)
+      SCAN (hash temp(m), build time: ?, time: ?, fetch: ?, ioread: ?, readrows: ?, rows: ?)
```

Comment:
It is necessary to check whether the answer sheet should be changed or the test case should be changed.

Reply:
in the testcase seems to check memory limit.
case line is 
select /*+ ordered */ count(*) from (select * from t1 limit 1) a, (select * from t1 limit 100000) b where a.a = b.a and a.b = b.b and a.c = b.c;

does this statement affect memory limit?